packages/test/test-version-utils/src/test/versionUtils.spec.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { getRequestedRange } from "../versionUtils";
+
+describe("Get the major version number above or below the baseVersion", () => {
+    // assert for major bumps
+    assert.strictEqual(getRequestedRange("1.0.0", -1), "^0.59.0-0");
+    assert.strictEqual(getRequestedRange("1.0.0", -2), "^0.58.0-0");
+    assert.strictEqual(getRequestedRange("1.0.0", 1), "^2.0.0-0");
+    assert.strictEqual(getRequestedRange("2.0.0", -1), "^1.0.0-0");
+
+    // assert for internal release
```

Comment:
@tylerbutler - this comment may not be directly related to this function, but since e2e tests are using the return value to query (npmjs, I believe) registry to determine what is available, do we need a work item around e2e handling these "previous" versions now possibly residing in internal registry?